mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -288,6 +294,21 @@ def run_create_tutorial(input_file):
         logger.exception(e)
 
 
+@click.command("archive")
+@click.option(
```

Comment:
we could use a similar approach here as in `run_generate_stats`. There we have `project_id_list` as an input.

Reply:
Will add support for lists as argument.